lightkurve/correctors/regressioncorrector.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, lc):
             raise ValueError('Input light curve has NaNs in `flux_err`. '
                              'Please remove NaNs before correction '
                              '(e.g. using `lc = lc.remove_nans()`).')
-        if np.any(lc.flux_err <= 0):
+        if np.any(lc.flux_err[np.isfinite(lc.flux_err)] <= 0):
```

Comment:
Surely this will never be needed if the ValueError above is always triggered first?

Reply:
We do allow *all* `flux_err` values to be NaN above, in which case this line was creating noisy RuntimeWarnings, hence the change!